src/green-web-checker.php

```diff
@@ -17,21 +17,28 @@ function tgwf_run_site_check() {
 	$greencheck = str_replace( 'https://', '', $greencheck );
 	$greencheck = explode( '/', $greencheck );
 
+	// Work out if the nocache flag is set or not, and prep it.
+	$cacheflag = '';
+
+	if ( isset( $_GET['nocache'] ) && ! empty( $_GET['nocache'] ) ) {
+		$cacheflag = '?' . $_GET['nocache'];
```

Comment:
One slight error here Han, this should be:

```php
		$cacheflag = '?nocache=' . $_GET['nocache'];
```

At the moment, the request that arrives at the API looks like this (it's missing the `nocache=`)
<img width="1350" height="54" alt="Image" src="https://github.com/user-attachments/assets/09441acb-5160-4410-98f7-3dda76436f8e" />